config/images.go

```diff
@@ -0,0 +1,68 @@
+	CONTAINER_IMAGE_PGO_CLIENT               = "pgo-client"
+	CONTAINER_IMAGE_PGO_LOAD                 = "pgo-load"
+	CONTAINER_IMAGE_PGO_RMDATA               = "pgo-rmdata"
+	CONTAINER_IMAGE_PGO_SQL_RUNNER           = "pgo-sqlrunner"
+	CONTAINER_IMAGE_CRUNCHY_ADMIN            = "crunchy-admin"
+	CONTAINER_IMAGE_CRUNCHY_BACKREST_RESTORE = "crunchy-backrest-restore"
+	CONTAINER_IMAGE_CRUNCHY_COLLECT          = "crunchy-collect"
+	CONTAINER_IMAGE_CRUNCHY_GRAFANA          = "crunchy-grafana"
+	CONTAINER_IMAGE_CRUNCHY_PGBADGER         = "crunchy-pgbadger"
+	CONTAINER_IMAGE_CRUNCHY_PGBENCH          = "crunchy-pgbench"
+	CONTAINER_IMAGE_CRUNCHY_PGBOUNCER        = "crunchy-pgbouncer"
+	CONTAINER_IMAGE_CRUNCHY_PGDUMP           = "crunchy-pgdump"
+	CONTAINER_IMAGE_CRUNCHY_PGRESTORE        = "crunchy-pgrestore"
+	CONTAINER_IMAGE_CRUNCHY_POSTGRES_HA      = "crunchy-postgres-ha"
+	CONTAINER_IMAGE_CRUNCHY_POSTGRES_GIS_HA  = "crunchy-postgres-gis-ha"
+	CONTAINER_IMAGE_CRUNCHY_PROMETHEUS       = "crunchy-prometheus"
+)
+
+// a map of the "RELATED_IMAGE_*" environmental variables to their defined
+// container image names, which allows certain packagers to inject the full
+// definition for where to pull a container image from
+//
+// See: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/contributors/design-proposals/related-images.md
+var RelatedImageMap = map[string]string{
+	"RELATED_IMAGE_PGO_BACKREST":             CONTAINER_IMAGE_PGO_BACKREST,
+	"RELATED_IMAGE_PGO_BACKREST_REPO":        CONTAINER_IMAGE_PGO_BACKREST_REPO,
+	"RELATED_IMAGE_PGO_BACKREST_REPO_SYNC":   CONTAINER_IMAGE_PGO_BACKREST_REPO_SYNC,
+	"RELATED_IMAGE_PGO_BACKREST_RESTORE":     CONTAINER_IMAGE_PGO_BACKREST_RESTORE,
+	"RELATED_IMAGE_PGO_CLIENT":               CONTAINER_IMAGE_PGO_CLIENT,
+	"RELATED_IMAGE_PGO_LOAD":                 CONTAINER_IMAGE_PGO_LOAD,
+	"RELATED_IMAGE_PGO_RMDATA":               CONTAINER_IMAGE_PGO_RMDATA,
+	"RELATED_IMAGE_PGO_SQL_RUNNER":           CONTAINER_IMAGE_PGO_SQL_RUNNER,
+	"RELATED_IMAGE_CRUNCHY_ADMIN":            CONTAINER_IMAGE_CRUNCHY_ADMIN,
+	"RELATED_IMAGE_CRUNCHY_BACKREST_RESTORE": CONTAINER_IMAGE_CRUNCHY_BACKREST_RESTORE,
+	"RELATED_IMAGE_CRUNCHY_COLLECT":          CONTAINER_IMAGE_CRUNCHY_COLLECT,
+	"RELATED_IMAGE_CRUNCHY_GRAFANA":          CONTAINER_IMAGE_CRUNCHY_GRAFANA,
+	"RELATED_IMAGE_CRUNCHY_PGBADGER":         CONTAINER_IMAGE_CRUNCHY_PGBADGER,
+	"RELATED_IMAGE_CRUNCHY_PGBENCH":          CONTAINER_IMAGE_CRUNCHY_PGBENCH,
+	"RELATED_IMAGE_CRUNCHY_PGBOUNCER":        CONTAINER_IMAGE_CRUNCHY_PGBOUNCER,
+	"RELATED_IMAGE_CRUNCHY_PGDUMP":           CONTAINER_IMAGE_CRUNCHY_PGDUMP,
+	"RELATED_IMAGE_CRUNCHY_PGRESTORE":        CONTAINER_IMAGE_CRUNCHY_PGRESTORE,
+	"RELATED_IMAGE_CRUNCHY_POSTGRES_HA":      CONTAINER_IMAGE_CRUNCHY_POSTGRES_HA,
+	"RELATED_IMAGE_CRUNCHY_POSTGRES_GIS_HA":  CONTAINER_IMAGE_CRUNCHY_POSTGRES_GIS_HA,
+	"RELATED_IMAGE_CRUNCHY_PROMETHEUS":       CONTAINER_IMAGE_CRUNCHY_PROMETHEUS,
```

Comment:
Just a note that the Grafana and Prometheus images are deployed via Ansible, and therefore will not be utilized directly by the Operator.